firebase-functions/src/androidTest/java/com/google/firebase/functions/TestInteropAppCheckTokenProvider.java

```diff
@@ -66,6 +66,12 @@ public Task<AppCheckTokenResult> getToken(boolean forceRefresh) {
     return Tasks.forResult(testToken);
   }
 
+  @NonNull
```

Comment:
Is the logic to have the functions SDK invoke the new interop method in a separate PR? If so, maybe move this change there as well?

Reply:
Separate PR, but this change needs to be here to allow this to build (since it implements the interface directly)